api/desecapi/urls/version_1.py

```diff
@@ -0,0 +1,61 @@
+from django.conf.urls import include, url
+from djoser.views import UserView
+from rest_framework.routers import SimpleRouter
+
+from desecapi import views
+
+
+tokens_router = SimpleRouter()
+tokens_router.register(r'', views.TokenViewSet, base_name='token')
+
+auth_urls = [
+    # Old user management
+    # TODO deprecated, remove
+    url(r'^users/create/$', views.UserCreateView.as_view(), name='user-create'),  # deprecated
+    url(r'^token/create/$', views.TokenCreateView.as_view(), name='token-create'),  # deprecated
+    url(r'^token/destroy/$', views.TokenDestroyView.as_view(), name='token-destroy'),  # deprecated
+
+    # New user management
+    url(r'^users/$', views.UserCreateView.as_view(), name='register'),
+
+    # Token management
+    url(r'^token/login/$', views.TokenCreateView.as_view(), name='login'),
+    url(r'^token/logout/$', views.TokenDestroyView.as_view(), name='logout'),
+    url(r'^', include('djoser.urls.authtoken')),  # note: this is partially overwritten by the two lines above
+    url(r'^tokens/', include(tokens_router.urls)),
+
+    # User home
+    url(r'^me/?$', UserView.as_view(), name='user'),
+]
+
+api_urls = [
+    # API home
+    url(r'^$', views.Root.as_view(), name='root'),
+
+    # Domain and RRSet endpoints
+    url(r'^domains/$', views.DomainList.as_view(), name='domain-list'),
+    url(r'^domains/(?P<name>[a-zA-Z\.\-_0-9]+)/$', views.DomainDetail.as_view(), name='domain-detail'),
+    url(r'^domains/(?P<name>[a-zA-Z\.\-_0-9]+)/rrsets/$', views.RRsetList.as_view(), name='rrsets'),
+    url(r'^domains/(?P<name>[a-zA-Z\.\-_0-9]+)/rrsets/(?P<subname>(\*)?[a-zA-Z\.\-_0-9=]*)\.\.\./(?P<type>[A-Z][A-Z0-9]*)/$', views.RRsetDetail.as_view(), name='rrset'),
+    url(r'^domains/(?P<name>[a-zA-Z\.\-_0-9]+)/rrsets/(?P<subname>[*@]|[a-zA-Z\.\-_0-9=]+)/(?P<type>[A-Z][A-Z0-9]*)/$', views.RRsetDetail.as_view(), name='rrset@'),
+
+    # DNS query endpoint
+    # TODO remove?
+    url(r'^dns$', views.DnsQuery.as_view(), name='dns-query'),
+
+    # DynDNS update endpoint
+    url(r'^dyndns/update$', views.DynDNS12Update.as_view(), name='dyndns12update'),
+
+    # Donation endpoints
+    url(r'^donation/', views.DonationList.as_view(), name='donation'),
+
+    # Unlock endpoints
+    url(r'^unlock/user/(?P<email>.+)$', views.unlock, name='unlock/byEmail'),
```

Comment:
I just realized that this causes PII to be logged (in this case, email address). This is considered bad practice. Also, there may be issues with URI encoding edge cases etc.

I created an issue for this (#161), but I don't think it needs to be in scope for launch.